### ISSUE TITLE
bin/setup-production doesn't create DATABASE_PATH directory

### ISSUE THREAD

**Issue by nickjer** [OSC/ood-myjobs#197]
Currently `DATABASE_PATH` is set in `.env.production` as:

```bash
DATABASE_PATH=$OOD_DATAROOT/production.sqlite3
```

and `bin/setup-production` will create the `OOD_DATAROOT`. But if this location changes to something else:

```bash
DATABASE_PATH=$HOME/my/production/dbs/production.sqlite3
```

then `bin/setup-production` crashes. So `bin/setup-production` should be updated to create the directory of `DATABASE_PATH` before trying to create the database.

**Comment by nickjer** [comment]
@brianmcmichael what is the progress of this?

**Comment by ericfranz** [comment]
Let me take care of this.

**Comment by basilgohar** [comment]
fixed by #244
